clarity/src/vm/docs/mod.rs

```diff
@@ -1912,8 +1912,8 @@ const STX_GET_BALANCE: SimpleFunctionAPI = SimpleFunctionAPI {
     signature: "(stx-get-balance owner)",
     description: "`stx-get-balance` is used to query the STX balance of the `owner` principal.
 
-This function returns the STX balance of the `owner` principal. In the event that the `owner`
```

Comment:
I think it could be appropriate to specify `1 STX = 1 million microstacks` under the top heading of `Functions`.

Reply:
Thanks Pavi. I think that would have to go into the [docs repo](https://github.com/stacks-network/docs), right?